outputs.tf

```diff
@@ -14,51 +14,50 @@
  * limitations under the License.
  */
 
-output "keyring" {
-  description = "Self link of the keyring."
-  value       = google_kms_key_ring.key_ring.id
-
-  # The grants are important to the key be ready to use.
-  depends_on = [
-    google_kms_crypto_key_iam_binding.owners,
-    google_kms_crypto_key_iam_binding.decrypters,
-    google_kms_crypto_key_iam_binding.encrypters,
-  ]
+output "existing_keyring" {
+  description = "Existing keyring is used, i.e. keyring has been created."
+  value       = var.existing_keyring
 }
 
-output "keyring_resource" {
-  description = "Keyring resource."
-  value       = google_kms_key_ring.key_ring
+output "keys" {
+  description = "Map of key name => id."
+  value       = { for key in local.crypto_keys : key.name => key.id }
+}
 
-  # The grants are important to the key be ready to use.
-  depends_on = [
```

Comment:
Can we keep the output dependency to make sure IAM is applied before this output is resolved?